messages/store.json

```diff
@@ -148,6 +148,18 @@
 
         "getEPN": {
             "cmdDescription": "Lists all EPN values"
+        },
+        "unmap": {
+            "cmdDescription": "Unmaps an extension from a store",
+            "StoreNameFlagDescription": "The name of the webstore to map the extension",
```

Comment:
would we consider creating a "flag" section which stores all the flag descriptions so that the same description is referenced in code. That way if we ever need to update it we can just do it from one place. We could also have one for "errors" which hold common errors throughout the extension commands.

Reply:
That's a good idea